src/components/Panel.tsx

```diff
@@ -1,45 +1,75 @@
+// "warning" (yellow), "danger" (red), "instruction" (blue), and "default" (gray).
+// You can also define your own styles.  If you do not pass in a value for the style prop,
+// the style will be set to "default".
+
+// There are two ways to populate the panel with content: you can use either the body prop,
+// which accepts a JSX element, or the text prop, which accepts an HTML string.
+
+// To make a static (as opposed to a collapsible) Panel, set the collapsible prop to false.
+// This will: remove the open/close icon; prevent the panel from opening/closing when the header
+// is clicked; render the header as a div rather than a button; and automatically ensure that the
+// panel is always open (i.e. you do not need to also set the openByDefault prop).
+
 export interface PanelOwnProps {
   style?: string;
   headerText: string;
-  body: JSX.Element;
+  body?: JSX.Element;
+  text?: string;
+  openByDefault?: boolean;
+  collapsible?: boolean;
 }
 
 export interface PanelState {
   display: string;
 }
 
 export default class Panel extends React.Component<PanelOwnProps, PanelState> {
-  constructor() {
-    super();
+  static defaultProps = {
+    style: "default",
+    collapsible: true,
+    openByDefault: false
+  };
+
+  constructor(props: PanelOwnProps) {
+    super(props);
     this.renderHeader = this.renderHeader.bind(this);
     this.toggle = this.toggle.bind(this);
-    this.state = { display: "collapse" };
+    let display = (this.props.openByDefault || !this.props.collapsible) ? "" : "collapse";
+    this.state = { display };
   }
 
-  toggle() {
+  toggle(e) {
+    e.preventDefault();
     let display = this.state.display === "collapse" ? "" : "collapse";
     this.setState({ display });
   }
 
   renderHeader(): JSX.Element {
-    let icon = this.state.display === "collapse" ? "down-icon" : "up-icon";
-    return (
-      <button className="panel-heading" onClick={this.toggle}>
-        <span className="panel-title">{this.props.headerText}</span>
-        <GenericWedgeIcon className={icon} />
-      </button>
+    let title = <span className="panel-title">{this.props.headerText}</span>;
+    let iconName = this.state.display === "collapse" ? "down-icon" : "up-icon";
+    let icon = this.props.collapsible ? <GenericWedgeIcon className={`${iconName} icon`} /> : null;
+
+    return React.createElement(
+      this.props.collapsible ? "button" : "div",
+      { className: "panel-heading", onClick: this.props.collapsible ? this.toggle : null },
+      [title, icon]
     );
   }
 
   render(): JSX.Element {
+    const staticPanel = !this.props.collapsible ? "static-panel" : "";
     return (
-      <li className={`panel panel-${this.props.style ? this.props.style : "default"}`}>
+      <div className={`panel panel-${this.props.style} ${staticPanel}`}>
         { this.renderHeader() }
-        <section className={`panel-body ${this.state.display}`}>{this.props.body}</section>
-      </li>
+        { this.props.text ?
```

Comment:
I think there's a better way to define this. It seems you require at least either `text` or `body`. If you don't have either one of them (I know that won't realistically happen but) then you'll render an empty `<section>` tag. I would say use `|` to define if a props has different value types, but you are rendering differently based on what they are.

You might be able to use typescript guards to render based on the type of prop you received. Might be worth looking into just for general knowledge, too.